### PR TITLE
feat(custom_messages): Custom messages are now always visible

### PR DESCRIPTION
**Pull request by jeromecambon**
- Custom messages do not scroll anymore, and are always visible.

Covers https://github.com/bonitasoft/bonita-documentation-site/issues/274

### CODE REVIEW

**Comment by jeromecambon** [comment]
For testing, comment out one of the custom message (see last commit)

**Comment by jeromecambon** [comment]
Tested on mobile (iPhone)

File: src/stylesheets/toolbar.scss [hunk 1]
```diff
@@ -70,3 +77,13 @@
...
   }
 }
+
+.out-of-support-block {
+  background-color: var(--color-admonition-important-bg);
+  border-color: var(--color-admonition-important);
+  color: var(--color-admonition-important-text);
+  text-align: center;
+  width:100%;
+}
```
Comment (tbouffard) [inline src/stylesheets/toolbar.scss:87]: ❓ This class is already defined in https://github.com/bonitasoft/bonita-documentation-theme/blob/d315394fd4414de15b8b2718b47407d51c17ad5e/src/stylesheets/doc.scss#L1092
Are we sure we need the duplication?

Comment (jeromecambon) [reply src/stylesheets/toolbar.scss:87]: Good catch !
I'll remove .out-of-support-block from doc.scss and move also .next-release-block in toolbar.scss

Comment (tbouffard) [reply src/stylesheets/toolbar.scss:87]: fixed in 5b8df92

**Review by tbouffard — APPROVED** [review]
Looks good.
The toc in the right side of the page is now displayed below its original position when there is no message. This is ok as a first implementation. We will see how we can improve this later. 

before | now
---- | ---------
image | image

**Pull request merged**